app/routers/metrics.py

```diff
@@ -1,7 +1,10 @@
-from typing import Annotated, Literal, List
 import fastapi
+from fastapi import HTTPException, Response
```

Comment:
I think you're already importing fastapi in a general way and you use more than three methods of this package. It is not necessary to reimport the individual packages, instead you can use fastapi.Response and fastapi.HTTPException to call the functions on lines 115 and 117

Reply:
Thanks for the suggestion!